example/handlers/adieu/index.jsx

```diff
@@ -0,0 +1,11 @@
+var React = require('react');
+var PageWrapper = require('../../components/page-wrapper.jsx');
+var Container = require('./container.jsx');
+
+module.exports = function adieuHandler (req, res) {
+	var RenderComponent = (<PageWrapper>
```

Comment:
A more legible format for this:

```javascript
var RenderComponent = (
  <PageWrapper>
    <Container name="Lorelai" />
  </PageWrapper>
);
```

See how it lines the starting and ending parts up so they are easily identifiable?  This helps others read your code.